### ISSUE TITLE
19 April Workshop

### ISSUE THREAD

**Issue by jialincheoh** [ironhacks/ironhacks-blog#20]
IronHacks is hosting a workshop on 19th April, 2023, inviting social scientists in studying humans in the cloud using online experiments and computational methods. 

Sign Up Here

**Comment by sbrunswi** [comment]
I want a test email sent via email.

**Comment by jialincheoh** [comment]
Link to pictures @sbrunswi

**Comment by ManthanKeim** [comment]
@all, sent, check spam if not in inbox.

**Comment by sbrunswi** [comment]
1) Date is 21
2) Lots of types
3) use the text from our google docs developed with howard underneath! So make more more substantial. So copy from the drive.
4) https://unsplash.com/photos/KgLtFCgfC28
5) Title on blog Invitation to join Computational liberal arts (CLA) workshop on IronHacks: Studying humans in the age of code, data, and AI using computational methods and online experiments
6) Find password for mailchimp @jialincheoh

**Comment by jialincheoh** [comment]
Please review @sbrunswi , so that @ManthanKeim can proceed with sending a test email

**Comment by sbrunswi** [comment]
I would like to edit the markdown right away. Did you fix the blog issue (hugo?)

**Comment by ManthanKeim** [comment]
about should be at the first

benefits should be at the end.

clean the writing a lot, and also order the issues 

benefits - bullet points or a visualization from jasmine( reference from aarhus workshop)

RSVP button at the end and beginning (Manthan)

**Comment by jialincheoh** [comment]
Regarding manthan's issue, I removed the "about section" and replaced with "benefits of using ironhacks for researchers", is that not a good idea?  @sbrunswi  , @ManthanKeim

**Comment by ManthanKeim** [comment]
Yellow color is not available in the add event, so used a solid blue color instead of red and removed the extra words in the RSVP button, [da351d4](https://github.com/ironhacks/ironhacks-blog/commit/da351d4f6346f5b40cf1c9b06fe2cdadfe954671) here.

**Comment by jasminebui1225** [comment]
I'm thinking of sth like this with event details attached to the image 
![Strategy Deck (3)](https://user-images.githubusercontent.com/116818449/229201697-ece69bde-77c1-4c27-82c1-3d7a73a122dc.png)

**Comment by ManthanKeim** [comment]
yes, we can also consider any other data science/coding image if we want.

**Comment by jasminebui1225** [comment]
- Connect the contact infor to website
- Update links
- Action items
- Computational liberal arts in the age of AI

**Comment by ManthanKeim** [comment]
Sent 

https://mailchi.mp/ef762650db07/computational-liberal-arts-cla-on-in-the-age-of-artificial-intelligence-ai

**Comment by ManthanKeim** [comment]
Reminder email with updated title,

Title Suggestion 1: Learn about IronHacks to perform computational research in liberal arts

Title Suggestion 2: Computational Liberal Arts (CLA) Workshop on IronHacks in the age AI

**Comment by jialincheoh** [comment]
Title 2 adjusted @sbrunswi  @jasminebui1225 @ManthanKeim 

Title Suggestion 2: Don't miss the Computational Liberal Arts (CLA) Workshop on IronHacks in the age AI

**Comment by jasminebui1225** [comment]
- [ ] Add Qualtrics survey (about lunch accommodation) to Mailchimp. Waiting for the link from Michael. 
- [ ] Should send out 1st Reminder by tomorrow afternoon.

**Comment by jasminebui1225** [comment]
@jialincheoh  @ManthanKeim  @harshpavuluri 
Here is the presentation slides for the workshop: https://docs.google.com/presentation/d/1XZT7J2bZ79en2r7F--auRd9hdyxRpCMd/edit?rtpof=true. 
- [ ] Please update the content. I can help with design layout later.

**Comment by ManthanKeim** [comment]
moved this from duplicate issue: 

@sbrunswi Dr. Brunswicker review: https://docs.google.com/document/d/1Z596APnbvzq8EpuBiIpNFHfkfDh5SIQ2Jgsmo-jEuQA/edit corrections to email template

**Comment by jialincheoh** [comment]
New agenda for April 21st workshop, we will only do till 2:30 pm. 



11:00 AM | Opening Remarks | Dr. Sorin Matei
-- | -- | --
11:05 AM | Introduction to IronHacks Platform | Dr. Sabine Brunswicker
11:20 AM | Research results presentation | Dr. Sabine BrunswickerJia Lin Cheoh
12:00 PM | Lunch Break |  
1:00 PM | IronHacks Fall 2022 Summary | Dr. Sabine BrunswickerFeature Winners (Lionel/Micks)
1:20 PM | Live demo of Capabilities of IronHacks PlatformFollowing along is encouraged | Dr. Sabine Brunswicker
2:00 PM | DiscussionQ&AFuture DevelopmentCollaboration Opportunities | Dr. Sabine BrunswickerRCODI Team Members
4:00 PM | Wrap Up and Concluding Remarks | Dr. Sabine BrunswickerDr. Sorin Matei

**Comment by sbrunswi** [comment]
11:00 Opening Remarks
11:05 Introduction to IronHacks Platform, and its features for research on computational liberal arts and examples of prior experiments performed with IronHacks
12: Live Demo of IronHacks Platform (following along encouraged), during working lunch
1:00 IronHacks Fall 2022 and Insights from Participant (Data Scientist)
1:20 Open Discussion & QA & Brainstorming on Collaboration Opportunities
2:20 Wrap-up of Formal Part of Workshop
2:30 Information Networking over Coffee & Snacks
4:00 pm End of Workshop

**Comment by sbrunswi** [comment]
@jialincheoh  and @ManthanKeim  - can you please update the blog, and then also the event website and the mailchimp email so that we can send out a reminder tomorrow?

**Comment by jialincheoh** [comment]
I do it when I arrive in lab early tomorrow, my internet is bloody slow now.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by jasminebui1225** [comment]
@jialincheoh  @ManthanKeim  can you guys update the blog? I updated the mailchimp already

**Comment by jialincheoh** [comment]
11:00 Opening Remarks
11:05 Introduction to IronHacks Platform, and its features for research on computational liberal arts and examples of prior experiments performed with IronHacks
12: Live Demo of IronHacks Platform (following along encouraged), during working lunch
1:00 IronHacks Fall 2022 and Insights from Participant (Data Scientist)
1:20 Open Discussion & QA & Brainstorming on Collaboration Opportunities
2:20 Wrap-up of Formal Part of Workshop
2:30 Informal Information Networking over Coffee & Snacks
